packages/sdk-react/src/components/receivables/InvoiceDetails/CreateReceivable/CreateReceivables.tsx

```diff
@@ -209,26 +209,24 @@ const CreateReceivablesBase = (props: InvoiceDetailsCreateProps) => {
               });
             })}
           >
-            <Stack spacing={2} sx={{ mt: 2 }}>
-              <Typography variant="h2" sx={{ mb: 2 }}>
-                {t(i18n)`Create Invoice`}
-              </Typography>
-              <Stack direction="column" spacing={4}>
-                <CustomerSection disabled={createReceivable.isPending} />
-                <EntitySection disabled={createReceivable.isPending} />
-                <ItemsSection
-                  defaultCurrency={settings?.currency?.default}
-                  actualCurrency={actualCurrency}
-                  onCurrencyChanged={setActualCurrency}
-                />
-                <PaymentSection disabled={createReceivable.isPending} />
-                <ReminderSection
-                  disabled={createReceivable.isPending}
-                  onUpdateOverdueReminder={onEditOverdueReminder}
-                  onUpdatePaymentReminder={onEditPaymentReminder}
-                  onCreateReminder={onCreateReminder}
-                />
-              </Stack>
+            <Typography variant="h1" sx={{ mb: 7 }}>
```

Comment:
It seems, the margin should be applied to the `<form/>` if we're talking about semantics

Reply:
@radist2s I moved the margin to form.